src/renderer/components/swap/utils.ts

```diff
@@ -0,0 +1,145 @@
+import {
```

Comment:
Add tests for `utils` -> https://github.com/thorchain/asgardex-electron/issues/388